### PR TITLE
Added SettingsLog.xml to allow end user to select what messages store in database

### PR DESCRIPTION
**Pull request by shawe**
Added support to persist miniLog messages to database
Added model and controllers to list and edit logs
Added new translations

### CODE REVIEW

**Comment by NeoRazorX** [comment]
No, bad idea to add database to minilog. Please remove database calls in minilog.
Also rename Log model to LogMessage.

File: Core/Controller/ListLogMessage.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        $values = [
+            ['code' => 'info', 'description' => $this->i18n->trans('type-info')],
+            ['code' => 'notice', 'description' => $this->i18n->trans('type-notice')],
+            ['code' => 'warning', 'description' => $this->i18n->trans('type-warning')],
+            ['code' => 'error', 'description' => $this->i18n->trans('type-error')],
+            ['code' => 'critical', 'description' => $this->i18n->trans('type-critical')],
+            ['code' => 'alert', 'description' => $this->i18n->trans('type-alert')],
+            ['code' => 'emergency', 'description' => $this->i18n->trans('type-emergency')]
+        ];
+        $selectValues = $this->codeModel::fromArray($values);
```
Comment (NeoRazorX) [inline Core/Controller/ListLogMessage.php]: This function is totally unnecesary.
Just do:
$this->addFilterSelect('ListLogMessage', 'level', 'type', 'level', $values);

File: Core/Model/CodeModel.php [hunk 2]
```diff
@@ -103,6 +103,24 @@ public static function all($tableName, $fieldCode, $fieldDescription, $addEmpty
...
     }
 
+    /**
+     * Load a CodeModel list (code and description) for the indicated array.
+     *
+     * @param array $array
+     *
+     * @return self[]
+     */
+    public static function fromArray($array = [])
```
Comment (NeoRazorX) [inline Core/Model/CodeModel.php]: Unnecesary function.

**Comment by shawe** [comment]
Solved

**Comment by shawe** [comment]
Solved again conflicts in en_EN.json

**Pull request merged**